htsohm/simulation/helium_void_fraction.py

```diff
@@ -37,8 +37,10 @@ def parse_output(output_file):
                 results['vf_helium_void_fraction'] = float(line.split()[4])
             except IndexError:
                 print()
-    print("\nVOID FRACTION :   %s\n" % (results['vf_helium_void_fraction']))
-
+    try:
+        print("\nVOID FRACTION :   %s\n" % (results['vf_helium_void_fraction']))
+    except KeyError:
+        print("\nERROR PARSING VOID FRACTION DATA.")
```

Comment:
got changed to:

        except (FileNotFoundError, IndexError, KeyError) as err:
            print(err)
            print(err.args)
            continue
        break